src/Extensions/Hosting.Services/Internal/ServiceInitializationEventSource.cs

```diff
@@ -31,7 +31,7 @@ public void LogHostBuildSucceded(int hostProcessId, string serviceType)
 				return;
 			}
 
-			LogHostBuildeSucceded(
+			LogHostBuildSucceded(
```

Comment:
```suggestion
			LogHostBuildSucceeded(
```

Reply:
Sorry I am actually wrong with this one.

Reply:
Renamed, thank you

Reply:
Oh, actually no, will push in a second

Reply:
should be fixed now, thanks